### PR TITLE
Add a new translation_arabic

### PR DESCRIPTION
**Pull request by LFakh**
added new Arabic translation

### CODE REVIEW

File: src/i18n/translations/ar.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    create_free_account: "إنشاء حساب مجاني",
+    create_shortcut: "تم تصميم المنتج",
+    current_password: "كلمة السر الحالية",
+    cut: 'إقطع',
+    date_modified: 'تغيير الوقت',
+    delete: 'حذف',
+    delete_permanently: "حذف دائم",
+    deploy_as_app: 'نشر كتطبيق',
+    descending: 'تنازلي',
+    desktop_background_fit: "خلفية سطح المكتب مناسبة"
```
Comment (AtkinsSJ) [inline src/i18n/translations/ar.js]: Missing a `,` at the end of this line.

File: src/i18n/translations/ar.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
+const de = {
```
Comment (AtkinsSJ) [inline src/i18n/translations/ar.js]: This needs to be named the same as the code:

```suggestion
const ar = {
```

File: src/i18n/translations/ar.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+        versions: "الإصدارات ",
+        yes_release_it: 'نعم أطلقه',
+        you_have_been_referred_to_puter_by_a_friend: "لقد تمت إحالتك إلى Puter بواسطة صديق!",
+        zip: "ملف مضغوط",
+        }
+};
+
+export default ar;
+    }
+};
```
Comment (AtkinsSJ) [inline src/i18n/translations/ar.js]: These two lines should be removed.

File: src/i18n/translations/ar.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        show_hidden: 'أظهر المخفي ',
+        sign_in_with_puter: "سجل الدخول ب Puter",
+        sign_up: "إشترك",
+        signing_in: "تسجيل الدخول",
+        size: 'الحجم',
+        sort_by: 'ترتيب حسب',
+        start: 'إبدأ ',
+        taking_longer_than_usual: 'يستغرق وقتا أطول من المعتاد ',
+        text_document: 'وثيقة نصية',
+        tos_fineprint: `بالنقر على "إنشاء حساب مجاني"، فإنك توافق على <a href="https://puter.com/terms" target="_blank">شروط الاستخدام</a> und der <a href="https://puter.com/privacy" target="_blank">حماية البيانات</a>من Puter ل`,
```
Comment (AtkinsSJ) [inline src/i18n/translations/ar.js]: "und der" here is German, please translate it.

File: src/i18n/translations/ar.js [hunk 5]
```diff
@@ -0,0 +1,150 @@
+const de = {
+    name: "Arab",
+    code: "ar",
+    dictionary: {
+    access_granted_to: "دخول مسموح به",
```
Comment (AtkinsSJ) [inline src/i18n/translations/ar.js]: Please indent lines 5-100 so it's clear they are part of the `dictionary`.

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
Hi, thank you for the translations!

I spotted a few issues. Please correct them.

**Comment by AtkinsSJ** [comment]
Also please add `ar` to `src/i18n/translations/translations.js`

**Comment by jelveh** [comment]
Thank you @LFakh for your contribution! I understand a little bit of Arabic so I translated one word as well haha

@AtkinsSJ Thanks for the review, I fixed the issues you raised.

**Pull request merged**